docs/user_guide/metrics.md

```diff
@@ -4,34 +4,40 @@ This doc contains all the metrics that Dragonfly components currently support. N
 
 ## Supernode
 
-- dragonfly_supernode_build_info{version, revision, goversion, arch, os} - build and version information of supernode.
-- dragonfly_supernode_http_requests_total{code, handler, method} - total number of http requests.
-- dragonfly_supernode_http_request_duration_seconds{code, handler, method} - http request latency in seconds.
-- dragonfly_supernode_http_request_size_bytes{code, handler, method} - http request size in bytes.
-- dragonfly_supernode_http_response_size_bytes{code, handler, method} - http response size in bytes.
-- dragonfly_supernode_peers{peer} - dragonfly peers, the label peer consists of the hostname and ip address of one peer.
-- dragonfly_supernode_tasks{cdnstatus} - dragonfly tasks.
-- dragonfly_supernode_tasks_registered_total{} - total times of registering new tasks. counter type.
-- dragonfly_supernode_dfgettasks{callsystem, status} - dragonfly dfget tasks.
-- dragonfly_supernode_dfgettasks_registered_total{callsystem} - total times of registering new dfgettasks. counter type.
-- dragonfly_supernode_dfgettasks_failed_total{callsystem} - total times of failed dfgettasks. counter type.
-- dragonfly_supernode_schedule_duration_milliseconds{peer} - duration for task scheduling in milliseconds.
-- dragonfly_supernode_cdn_trigger_total{} - total times of triggering cdn. counter type.
-- dragonfly_supernode_cdn_trigger_failed_total{} - total failed times of triggering cdn. counter type.
-- dragonfly_supernode_cdn_cache_hit_total{} - total times of hitting cdn cache. counter type.
-- dragonfly_supernode_cdn_download_total{} - total times of cdn downloading. counter type.
-- dragonfly_supernode_cdn_download_failed_total{} - total failure times of cdn downloading. counter type.
-- dragonfly_supernode_pieces_downloaded_size_bytes_total{} - total size of pieces downloaded from supernode in bytes. counter type.
-- dragonfly_supernode_gc_peers_total{} - total number of peers that have been garbage collected. counter type.
-- dragonfly_supernode_gc_tasks_total{} - total number of tasks that have been garbage collected. counter type.
+Name                                                                     | Type      | Description
+:----------------------------------------------------------------------- | :-------- | :----------
+dragonfly_supernode_build_info{version, revision, goversion, arch, os}   | gauge     | build and version information of supernode.
```

Comment:
I hope we can separate the metric name and its labels. So please add another column called labels